### PR TITLE
Trying stuff for WC connection..v2

### PR DESCRIPTION
**Pull request by Velenir**
Better workaround
Please test

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr267--dexreact.review.gnosisdev.com

File: src/api/wallet/WalletApiImpl.ts [hunk 1]
```diff
@@ -4,6 +4,104 @@ import assert from 'assert'
 import { getDefaultProvider } from '../'
 
 import WalletConnectProvider from '@walletconnect/web3-provider'
+// import WalletConnectQRCodeModal from '@walletconnect/qrcode-modal'
+;(window as any).WalletConnectProvider = WalletConnectProvider
+
+const oldEnable = WalletConnectProvider.prototype.enable
```
Comment (anxolin) [inline src/api/wallet/WalletApiImpl.ts]: I would be OK keeping the workaround, but we need an issue to remove it, and also open an issue to WC.

If this works I'm ok with it. Testing now.

File: src/api/wallet/WalletApiImpl.ts [hunk 2]
```diff
@@ -4,6 +4,104 @@ import assert from 'assert'
...
+WalletConnectProvider.prototype.enable = async function(): Promise<[string]> {
+  const accounts = await oldEnable.call(this)
+  this.wc.session && this.updateState(this.wc.session)
+  return accounts
+}
+
+// class ETHProv extends WalletConnectProvider {
```
Comment (anxolin) [inline src/api/wallet/WalletApiImpl.ts]: pls remove

File: src/api/wallet/WalletApiImpl.ts [hunk 3]
```diff
@@ -109,9 +207,11 @@ const subscribeToBlockchainUpdate = ({
 
 const wcOptions: WalletConnectInits = {
   package: WalletConnectProvider,
+  // package: ETHProv,
```
Comment (anxolin) [inline src/api/wallet/WalletApiImpl.ts]: pls remove

**Comment by Velenir** [comment]
> The effect is the loading is loading forever

I don't follow

**Comment by anxolin** [comment]
> I don't follow

1. You send a tx
2. Loading spins
3. I get the error I send in this and the other PR (in the console). The app doesn't show any error to the user, nor stops the "loading"

--> Effect: There's an error, the user is not made aware, the loader spins forever

**Comment by Velenir** [comment]
Closed in favor of #285

**Pull request closed without merging**